src/http/sr-modules/sr-modules.e2e-spec.ts

```diff
@@ -197,6 +193,16 @@ describe('SRModulesController (e2e)', () => {
         expect(resp.status).toEqual(425);
         expect(resp.body).toEqual({ message: 'Too early response', statusCode: 425 });
       });
+
+      it('should return 400 error if module_id is not a contract address or number', async () => {
```

Comment:
wrong section ('too early response case'). May be I've already seen fix for this in another PR

Reply:
now even cant understand what is wrong